src/cbmc/cbmc_parse_options.cpp

```diff
@@ -536,6 +536,11 @@ int cbmc_parse_optionst::doit()
     goto_model.validate(ns, validation_modet::INVARIANT);
   }
 
+  if(cmdline.isset("validate-ssa-equation"))
+  {
+    options.set_option("validate-ssa-equation", true);
+  }
```

Comment:
This should best be put in `cbmc_parse_optionst::get_command_line_options()`

Reply:
Done.